utils/utils.go

```diff
@@ -157,6 +157,55 @@ func CustomizeLibertyAnnotations(pts *corev1.PodTemplateSpec, la *websphereliber
 	pts.Annotations = rcoutils.MergeMaps(pts.Annotations, libertyAnnotations)
 }
 
+func CustomizeLicenseAnnotations(pts *corev1.PodTemplateSpec, la *webspherelibertyv1.WebSphereLibertyApplication) {
+	source := la.Spec.License.ProductEntitlementSource
+	pts.Annotations["productID"] = "87f3487c22f34742a799164f3f3ffa78"
+	pts.Annotations["productChargedContainers"] = "app"
+
+	metricValue := "VIRTUAL_PROCESSOR_CORE"
+	if la.Spec.License.Metric == webspherelibertyv1.LicenseMetricPVU && source == webspherelibertyv1.LicenseEntitlementNone || source == webspherelibertyv1.LicenseEntitlementFamilyEdition {
+		metricValue = "PROCESSOR_VALUE_UNIT"
+	}
+	pts.Annotations["productMetric"] = metricValue
+
+	productName := ""
```

Comment:
Can simplify this code by use `var productName LicenceEdition` so you can directly assign `la.spec.License.Edition` or  just do `pts.Annotations["productName] = string(la.spec.License.Edition)`

Reply:
@arturdzm Thanks for the review. Addressed the comment. Made few naming changes and updates to utils. New changes are in the last commit. Please review.